### DISCUSSION TITLE
only download in mp3

### DISCUSSION THREAD

**Discussion by skijbal** [RandomNinjaAtk/docker-lidarr-extended#131] [Q&A] [answered]
I only want to download in 128/mp3.  I don't care about lossless formats.  Is there a way to set this up to do that?  This is what I have setup.

- audioFormat=mp3
- audioBitrate=128
- requireQuality=true
- enableReplaygainTags=true

It won't download anything on a free account.  If I signup for a trial then it will download the flac and convert to 128/mp3.

Edit:
I answered my own question.

- audioFormat=native
- audioBitrate=low

**Comment by RandomNinjaAtk** [comment, marked as answer]
Yup you figured it out, by using these settings, like you stated:

    audioFormat=native
    audioBitrate=low
